core/src/main/scala/org/apache/spark/ui/jobs/StagePage.scala

```diff
@@ -472,12 +512,12 @@ private[ui] class StagePage(parent: StagesTab) extends WebUIPage("stage") {
         }}
         {if (hasInput) {
           <td sorttable_customkey={inputSortable}>
-            {inputReadable}
+            {s"$inputReadable / $inputRecords"}
```

Comment:
I worry that it would be confusing to a user if they see 0 because that is valid number of num records read/written. but maybe that is better than seeing -45.